src/Mjolnir.ts

```diff
@@ -650,10 +650,12 @@ export class Mjolnir {
         return await this.client.doRequest("POST", endpoint);
     }
 
-    public async shutdownSynapseRoom(roomId: string): Promise<any> {
-        const endpoint = `/_synapse/admin/v1/shutdown_room/${roomId}`;
-        return await this.client.doRequest("POST", endpoint, null, {
+    public async shutdownSynapseRoom(roomId: string, message?: string): Promise<any> {
+        const endpoint = `/_synapse/admin/v1/rooms/${roomId}`;
+        return await this.client.doRequest("DELETE", endpoint, null, {
             new_room_user_id: await this.client.getUserId(),
+            block: true,
+            message: message || "This room was shutdown by a moderator"
```

Comment:
we should exclude the message so we can continue using the default

Reply:
I don't understand this comment. Should I remove `"This room was shutdown by a moderator"`?

Reply:
yea, just so the default message from Synapse can be used.